spec/kibana_spec.rb

```diff
@@ -1,24 +1,9 @@
 # encoding: utf-8
 require 'spec_helper'
 
-kibana_version = '4.4.1'
-
-describe file("/opt/kibana-#{kibana_version}-linux-x64.tar.gz") do
-  it { should exist }
-  it { should be_file }
-end
-
-describe file("/opt/kibana-#{kibana_version}-linux-x64") do
-  it { should exist }
-  it { should be_directory }
-  its('owner') { should eq 'www-data' }
-  its('group') { should eq 'www-data' }
-end
-
 describe file('/opt/kibana') do
   it { should exist }
-  it { should be_symlink }
-  it { should be_linked_to "/opt/kibana-#{kibana_version}-linux-x64" }
+  it { should be_directory }
```

Comment:
Let's do some sanity-checking on owner/mode here. The previous tasks in the role were enforcing www-data since we're serving straight out of the directory—do we need to do the same here?